pages/404/index.tsx

```diff
@@ -0,0 +1,15 @@
+import Image from 'next/image';
+import error404 from "../../public/images/error404.svg";
+
+const FourOFour = () => {
+	return (
+		<div className={"bg-[#C80F2E] h-screen flex justify-center items-center " }>
+			<Image
+                src={error404}
+                alt="error404svg"
```

Comment:
**suggestion:** update "error404svg" to "Page Not Found"
**note:** `alt` should be meaningful